alembic.ini

```diff
@@ -0,0 +1,85 @@
+# A generic, single database configuration.
```

Comment:
We generated this root alembic.ini so that we can run migrations and seed the db using alembic CLI